flatcar-postinst

```diff
@@ -328,6 +328,25 @@ function patch_grub {
 
 patch_grub
 
+# Keep old nodes on cgroup v1
+if ! grep -q systemd.unified_cgroup_hierarchy "${OEM_MNT}/grub.cfg"; then
+    # if there is an explicit setting, don't touch it. Otherwise...
+    if [[ "${VERSION_ID}" != *"nightly"* ]]; then
```

Comment:
```suggestion
    if [[ "${BUILD_ID}" != "dev-"* ]]; then
```

Reply:
The `VERSION_ID` is always x.y.z but the `BUILD_ID` is used to encode dev builds - at least on Jenkins, I didn't check what manual local builds are using.

Reply:
I'll do it the way you suggest, seems safest. Local builds appear to just have a date in BUILD_ID, but they also don't get updated so it doesn't really matter.

For what its worth, the other alternative i was considering was:
```bash
if [[ "${VERSION_ID}" == "${VERSION}" ]]; then
```
as that seems to distinguish releases from developer/nightly builds.

Reply:
Thanks for checking, the releases also have the data in the BUILD_ID.

I guess for local developer builds it's ok to execute this code as they will have a proper version not based on the data like the nightly.

Reply:
either
```suggestion
    if [[ "${BUILD_ID}" != "dev-"* ]]; then
```
or 
```suggestion
    if [[ "${VERSION_ID}" == "${VERSION}" ]]; then
```
both are fine